app/src/main/java/org/simple/clinic/user/User.kt

```diff
@@ -138,6 +142,48 @@ data class User(
     UNAUTHORIZED;
 
     class RoomTypeConverter : RoomEnumTypeConverter<LoggedInStatus>(LoggedInStatus::class.java)
+
+  }
+
+  @Parcelize
+  data class Capabilities(val canTeleconsult: CapabilityStatus) : Parcelable
+
+  sealed class CapabilityStatus : Parcelable {
```

Comment:
Let's override `toString` for the sealed classes types since we will be using it as an enum.